src/components/modal/AposModal.vue

```diff
@@ -255,7 +255,7 @@ export default {
   }
 
   .apos-modal__main--with-rails {
-    grid-template-columns: 22% 60% 18%;
+    grid-template-columns: 22% 1fr minmax(250px, 18%);
```

Comment:
250 is based off the width of the widest common right rail component (the publishing boolean) plus sidebar padding. This might deserve adjustment, but it's partially mean as an introduction of some additional responsiveness in our grids.